brainstormforce/ultimate-addons-for-gutenberg
# Post Grid - Acceptance Testing Round 2

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/120ab759-bb2c-40b2-a5f4-942908090d72?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- Added Screencast in task.

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop the Post grid.
- Click on the edit icon and check layout looks good or not.
- Check the Style tab and the first tab should be the Layout Setting tabs
- Check taxonomy style option Normal/Highlighted.
- Check taxonomy style option Normal/Highlighted when image position is background (editor and frontend).
- Check by clicking on the Category link when Taxonomy is highlighted go to category archive or not.

### Checklist:
